sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -805,6 +805,8 @@ def download_blob(self, offset=None, length=None, **kwargs):
             The number of parallel connections with which to download.
         :keyword str encoding:
             Encoding to decode the downloaded bytes. Default is None, i.e. no decoding.
+        :keyword bool decompress:
+            This is True by default. Set it to False to disable the blob content auto-decompress.
```

Comment:
Let's improve the description by mentioning the terms "encoded/zipped files" or something along those lines